src/MapUpMap.js

```diff
@@ -1,21 +1,40 @@
 import React from 'react';
 import Map from 'react-map-gl';
-
+import Footer from './Footer';
 // eslint-disable-next-line import/no-webpack-loader-syntax
 import maplibregl from "!maplibre-gl";
-
+import axios from "axios";
 function MapUpMap() {
+
+  let apiKey = "MHc0JQm8IOI3imNzjG0nx74AymP5P6eZ";
+  let serviceUrl = 'https://api.os.uk/maps/vector/v1/vts';
+
+  // BRANDON TEST AXIOS
+  async function find() {
```

Comment:
In general, API code should go into some kind of dedicated code library file. In this case though I wouldn't put this into the main repository yet as it's only for debugging purposes (plus we can't use the API directly as we can't store the API key in the client app).

Reply:
As above, I'd remove this whole block as it's not 'doing' anything. It's also not related to adding a header/footer and it's good practice to have one feature per branch/pull request.